jib-core/src/main/java/com/google/cloud/tools/jib/cache/CacheDirectoryCreationException.java

```diff
@@ -14,15 +14,23 @@
  * the License.
  */
 
-package com.google.cloud.tools.jib.frontend;
+package com.google.cloud.tools.jib.cache;
 
-import com.google.cloud.tools.jib.cache.Cache;
 import java.nio.file.Path;
 
-/** Thrown when a directory could not be created to be used as the {@link Cache}. */
+/**
+ * Thrown when a directory could not be created to be used as the {@link
```

Comment:
I think the meaning is more in terms of failure to create a directory for use as a cache directory rather than failure to create a cache directory (which may imply a special type of directory).

Reply:
Hm I see. Either way the sentence is a bit hard to read